### PR TITLE
Adding CNV into the Metal CPT

### PR DESCRIPTION
**Pull request by jtaleric**
- Day2 install CNV

TODO - We might need to wait for CNV to be fully installed

### CODE REVIEW

**Comment by jtaleric** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-metal-dev-latest-x86-data-path-cnv-6nodes

**Comment by jtaleric** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-metal-dev-latest-x86-data-path-cnv-6nodes

**Comment by jtaleric** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-metal-dev-latest-x86-data-path-cnv-6nodes

**Comment by jtaleric** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-metal-dev-latest-x86-data-path-cnv-6nodes

**Comment by jtaleric** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-metal-dev-latest-x86-data-path-cnv-6nodes

**Comment by jtaleric** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-metal-dev-latest-x86-data-path-cnv-6nodes

**Comment by jtaleric** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-metal-dev-latest-x86-data-path-cnv-6nodes

**Comment by jtaleric** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-metal-dev-latest-x86-data-path-cnv-6nodes

**Comment by jtaleric** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-metal-dev-latest-x86-data-path-cnv-6nodes

**Comment by jtaleric** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-metal-dev-latest-x86-data-path-cnv-6nodes

**Comment by jtaleric** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-metal-dev-latest-x86-data-path-cnv-6nodes

**Comment by jtaleric** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-metal-dev-latest-x86-data-path-cnv-6nodes

File: ci-operator/step-registry/openshift-qe/installer/bm/day2/cnv/openshift-qe-installer-bm-day2-cnv-commands.sh [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+cat << EOF| oc apply -f -
+apiVersion: hco.kubevirt.io/v1beta1
+kind: HyperConverged
+metadata:
+  name: kubevirt-hyperconverged
+  namespace: openshift-cnv
+Spec: {}
+EOF
+
+sleep 20
```
Comment (rsevilla87) [inline ci-operator/step-registry/openshift-qe/installer/bm/day2/cnv/openshift-qe-installer-bm-day2-cnv-commands.sh:88]: Do we want to use infra nodes to host CNV stuff?

Comment (jtaleric) [reply ci-operator/step-registry/openshift-qe/installer/bm/day2/cnv/openshift-qe-installer-bm-day2-cnv-commands.sh:88]: which aspects? Does the product mention to do that?

**Comment by jtaleric** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-metal-dev-latest-x86-data-path-cnv-6nodes

**Comment by jtaleric** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-metal-dev-latest-x86-data-path-cnv-6nodes

**Comment by jtaleric** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-metal-dev-latest-x86-data-path-cnv-6nodes

File: ci-operator/step-registry/openshift-qe/installer/bm/day2/cnv/openshift-qe-installer-bm-day2-cnv-commands.sh (hunk 1 above)
Comment (jtaleric) [reply ci-operator/step-registry/openshift-qe/installer/bm/day2/cnv/openshift-qe-installer-bm-day2-cnv-commands.sh:88]: @jeniferh wdyt? is it worth it? 
```Optional: Configure Infra and Workloads node placement options for OpenShift Virtualization components.```

I don't see what they recommend moving. Things like the cni-linux-bridge, virt handler, bridge-marker, etc are per-node. Is the virtcontroller/virt-apiserver heavy components?

**Comment by jtaleric** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-metal-dev-latest-x86-data-path-cnv-6nodes

File: ci-operator/step-registry/openshift-qe/installer/bm/day2/cnv/openshift-qe-installer-bm-day2-cnv-commands.sh (hunk 1 above)
Comment (jeniferh) [reply ci-operator/step-registry/openshift-qe/installer/bm/day2/cnv/openshift-qe-installer-bm-day2-cnv-commands.sh:88]: Virt control plane load shouldn't be much of a concern unless you were just constantly churning VM creation, even then it tends to run on beefy baremetal nodes, don't really see Infra usage in practice.

**Comment by jtaleric** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-metal-dev-latest-x86-data-path-cnv-6nodes

**Comment by jtaleric** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-metal-dev-latest-x86-data-path-cnv-6nodes

**Comment by jtaleric** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-metal-dev-latest-x86-data-path-cnv-6nodes

**Comment by jtaleric** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-metal-dev-latest-x86-data-path-cnv-6nodes

**Comment by jtaleric** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-metal-dev-latest-x86-data-path-cnv-6nodes

**Comment by jtaleric** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-metal-dev-latest-x86-data-path-cnv-6nodes

**Comment by jtaleric** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-metal-dev-latest-x86-data-path-cnv-6nodes

**Comment by jtaleric** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-metal-dev-latest-x86-data-path-cnv-6nodes

**Pull request merged**